hack/lib/cmd.sh

```diff
@@ -198,6 +198,14 @@ function os::cmd::internal::expect_exit_code_run_grep() {
     fi
     local test_succeeded=$( ${test_eval_func} "${test_result}"; echo $? )
 
+    if (( ! test_succeeded )); then
+        os::text::print_blue "[DEBUG] Output content test failed. Debugging information follows:"
+        os::text::print_blue "[DEBUG] \${grep_args}=${grep_args}"
+        os::text::print_blue "[DEBUG] \${test_result}=${test_result}"
+        os::text::print_blue "[DEBUG] \${test_eval_func}=${test_eval_func}"
+        os::text::print_blue "[DEBUG] \${test_succeeded}=${test_succeeded}"
+    fi
```

Comment:
Jenkins output is still some weird pseudo-tty so we don't seem to get color there, but `ginkgo` somehow is allowed to do it ... oh well.